app/build.gradle

```diff
@@ -84,7 +84,18 @@ android {
 }
 
 dependencies {
+    constraints {
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.0") {
+            because("kotlin-stdlib-jdk7 is now a part of kotlin-stdlib")
+        }
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.0") {
+            because("kotlin-stdlib-jdk8 is now a part of kotlin-stdlib")
+        }
+    }
 // AndroidX
+    implementation 'androidx.activity:activity:1.7.1'
+    implementation 'androidx.activity:activity-compose:1.7.1'
+    implementation 'androidx.activity:activity-ktx:1.7.1'
```

Comment:
Are these new dependencies necessary? `androidx.activity:activity:1.7.1` is according to the documentation, but I don't see `-compose` and `-ktx` mentioned as necessary.

Reply:
When I click the link of the documentation then appear that 3 libraries but I tried using only  `androidx.activity:activity:1.7.1` and is working too. so let me remove the other ones.